### PR TITLE
feat: add e2e test for serverless plugin

### PR DESCRIPTION
**Pull request by Chever-John**
### Type of change:

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

Add e2e test for serverless plugin!

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: test/e2e/scaffold/k8s.go [hunk 1]
```diff
@@ -554,3 +555,70 @@ func (s *Scaffold) GetKubernetesClient() *kubernetes.Clientset {
 	assert.Nil(ginkgo.GinkgoT(), err, "get kubernetes client")
 	return client
 }
+
+func (s *Scaffold) CreateApisixRouteByChever(routeID string, body []byte) error {
```
Comment (tao12345666333) [inline test/e2e/scaffold/k8s.go:559]: what does `ByChever` mean?

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: test/e2e/scaffold/k8s.go (hunk 1 above)
Comment (Chever-John) [reply test/e2e/scaffold/k8s.go:559]: Just for demo, I found there has another contributor has write this same function. 
So I  want to use his code after his PR was merged.

**Comment by tao12345666333** [comment]
You can just create one ApisixRoute resource, and enable the serverless plugin.
Then try to check the route and response.

File: test/e2e/suite-plugins/serverless.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ - name: rule1
+   match:
+     hosts:
+     - httpbin.org
+     paths:
+       - /ip
+   backends:
+   - serviceName: %s
+     servicePort: %d
+     weight: 10
```
Comment (tao12345666333) [inline test/e2e/suite-plugins/serverless.go:55]: We don't need this line

File: test/e2e/suite-plugins/serverless.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
```
Comment (tao12345666333) [inline test/e2e/suite-plugins/serverless.go:91]: Ditto

**Review by tao12345666333 — COMMENTED** [review]
Thanks

File: test/e2e/suite-plugins/serverless.go (hunk 2 above)
Comment (Chever-John) [reply test/e2e/suite-plugins/serverless.go:55]: Got it.

File: test/e2e/suite-plugins/serverless.go (hunk 3 above)
Comment (Chever-John) [reply test/e2e/suite-plugins/serverless.go]: Done.

Comment (Chever-John) [reply test/e2e/suite-plugins/serverless.go]: Done

**Review by tao12345666333 — APPROVED** [review]
Thanks

LGTM

**Review by leslie-tsang — APPROVED** [review]
LGTM

**Pull request merged**